examples/quickstart/wadm-values.yaml

```
@@ -0,0 +1,6 @@
+wadm:
+  image:
+    tag: v0.12.2
+  config:
+    nats:
+      server: "nats-cluster.default.svc.cluster.local:4222"
```

Updated for the deployment instructions change:
```suggestion
      server: "nats.default.svc.cluster.local:4222"
```